.chloggen/splunk_enforce_metrics_naming_regex.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: internal/splunk
```

This is only applicable to the hec receiver, right? Let's make it more user-friendly

```suggestion
component: receiver/splunkhec
```